lib/checkunusedfunctions.cpp

```diff
@@ -245,11 +294,13 @@ bool CheckUnusedFunctions::check(ErrorLogger * const errorLogger, const Settings
             it->first == "if")
             continue;
         if (!func.usedSameFile) {
-            std::string filename;
-            if (func.filename != "+")
-                filename = func.filename;
-            unusedFunctionError(errorLogger, filename, func.lineNumber, it->first);
-            errors = true;
+            if (!isOperatorFunction(it->first)) {
```

Comment:
does it work with an conditional continue here instead?

    if (isOperatorFunction(it->first))
        continue;

I prefer early return/continue/etc.